resources/eventing/charts/nats/templates/10-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.global.install.enabled }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nats-operator
+  # Change to the name of the namespace where to install NATS Operator.
+  # Alternatively, change to "nats-io" to perform a cluster-scoped deployment in supported versions.
+  namespace: {{ .Values.global.namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: nats-operator
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        name: nats-operator
+    spec:
+      serviceAccountName: nats-operator
+      containers:
+      - name: nats-operator
+        image: connecteverything/nats-operator:0.7.4
```

Comment:
Let's sync this image to the Kyma registry and use it: https://github.com/kyma-project/test-infra/blob/master/development/image-syncer/external-images.yaml

Reply:
Will be solved in https://github.com/kyma-project/kyma/issues/9875